src/utilities/api/interactionManager.ts

```diff
@@ -118,6 +118,11 @@ export const updateVoucher = async (id: string, amount: number) =>
     .then((res) => res)
     .catch((err) => err);
 
+export const getMerchantGiftCards = async (seller_id: string, secret: string) =>
+  axios
+    .get(sellers + seller_id + '/gift_cards/' + secret)
+    .then((res) => res) // don't catch error, throw it up the stack
```

Comment:
:puke: 😂

Reply:
😂  I wanted to differentiate from all the other requests that have a catch block